src/ResponsiveImages/ResponsiveImageGenerator.php

```diff
@@ -103,7 +105,14 @@ public function generateResponsiveImage(
 
         $responsiveImageHeight = ImageFactory::load($tempDestination)->getHeight();
 
-        $finalImageFileName = $this->appendToFileName($responsiveImagePath, "_{$responsiveImageHeight}");
+        // the user can customize the name like he wants, be we expect the last part in a certain format
```

Comment:
Avoid `he` or `she` in comments. You could use plural form. "Users can customise the name they want...", or use passive form: "The name can be customized..."